ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml

```diff
@@ -108,7 +112,7 @@ tests:
   container:
     from: src
 - as: govet
-  commands: IS_CONTAINER=TRUE make vet
+  commands: IS_CONTAINER=TRUE GO111MODULE=auto make vet
```

Comment:
is it possible these will conflict?
https://github.com/openshift/kubernetes-autoscaler/blob/master/Makefile#L14